### ISSUE TITLE
QR format

### ISSUE THREAD

**Issue by Zambi9** [lincolnloop/python-qrcode#150]
When I use make() with a bitcoin address, for example "18cSPbDduAcAds7xr8tn4LUsQNwJ6iCHUy", I get a QR image very different from the one produced in blockchain.info (https://blockchain.info/address/18cSPbDduAcAds7xr8tn4LUsQNwJ6iCHUy)

I tried the four error correction constants.

Any idea how can I generate a QR image consistent with theirs?

**Comment by heuer** [comment]
What's your problem? 

The generated QR Code can be read by usual readers with the correct input string.

These QR codes should provide the same result:

qrode

```python

>>> import qrcode
>>> qrcode.make('18cSPbDduAcAds7xr8tn4LUsQNwJ6iCHUy').show()
```

![qrcode-bitcoin](https://user-images.githubusercontent.com/364926/37806199-12d27e04-2e3f-11e8-81ad-d850a34115fe.png)


segno

```python

>>> import segno
>>> segno.make('18cSPbDduAcAds7xr8tn4LUsQNwJ6iCHUy').show()
```
![segno-bitcoin](https://user-images.githubusercontent.com/364926/37806237-36b1756e-2e3f-11e8-88ba-0e2a5d7a078a.png)

**Comment by SmileyChris** [comment]
Like has been stated already, it doesn't matter.

But the technical reason that it's a different image is the decision on which is the best mask to overlay over the data. Here's the matching image using the same mask (6) that blockchain.info decides on.

```python
from qrcode import QRCode
q = QRCode(error_correction=1)
q.add_data('18cSPbDduAcAds7xr8tn4LUsQNwJ6iCHUy')
q.best_fit()
q.makeImpl(False, 6)
q.make_image().show()
```

**Comment by Zambi9** [comment]
From programmer's or functional perspective, it really doesn't matter.
But from average user's perspective, it is nice to have a result that is consistent with some kind of well known source. Be it blockchain.info, bitaddress.org or something similar (mind you, those two aren't consistent with each other either).

With error correction = 1 and mask = 6 the image is indeed identical to the one in blockchain.info, but it doesn't work with other addresses I tried. I then tried to check all mask, error correction combinations. For example:

```
addr = "1JHAz5VFfMuXttrs8TCaSGagG2jxeMd7Za"
for mask in range(0,8):
    for ec in range (0,4):
        print ("Mask: ",mask," Err Corr: ",ec)
        q = qrcode.QRCode(error_correction=ec)
        q.add_data(addr)
        q.best_fit()
        q.makeImpl(False, mask)
        q.print_ascii()
```
and none of the generated images are identical to what I get in blockchain.info or bitaddress.org

any ideas?
